portfolio/src/main/java/com/google/sps/servlets/MusicalListServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/random-album")
+public class MusicalListServlet extends HttpServlet {
+
+    ArrayList<String> faveMusicals;
```

Comment:
Nit: Can be initialized as List faveMusicals = new ArrayList<>(Arrays.asList("Hamilton", ...)) instead of doing this for every function call.